package.json

```diff
@@ -27,8 +27,10 @@
     "iconv-lite": "0.4.13",
     "minimist": "^1.2.0",
     "native-keymap": "0.1.2",
+    "pty.js": "git+https://github.com/jeremyramin/pty.js.git#28f2667",
     "sax": "1.1.2",
```

Comment:
@jrieken when I did try out this branch, our changes with NPM 3 were not present, so I ended up getting an older version of NAN for the terminal. Since you merged, I would assume, this terminal picks up the latest NAN we provide and it should be fine. As long as you can run the terminal successfully with latest, it should be good.